modules/azure-public-dns/cname_record.tf

```diff
@@ -6,5 +10,7 @@ resource "azurerm_dns_cname_record" "this" {
 
   name   = var.cname_records[count.index].name
   ttl    = var.cname_records[count.index].ttl
-  record = var.cname_records[count.index].record
-}
\ No newline at end of file
+  record = var.cname_records[count.index].shutter == true ? "hmcts-${replace(split(".", var.cname_records[count.index].name)[length(split(".", var.cname_records[count.index].name))-1], "www", split(".", var.zone_name)[0])}-shutter-${var.env}.${local.shutter_domain}" : var.cname_records[count.index].record
```

Comment:
this seems more complicated than what's in azure-platform-terraform, is there a reason for that?
`${split(".", replace(var.shutter_apps[count.index], "www.", ""))[0]`
https://github.com/hmcts/azure-platform-terraform/blob/master/modules/shutter_page/cdn_endpoints.tf#L3

Reply:
yeah, because in azure-platform-terraform we have the input as `www.apply-divorce.service.gov.uk` The code there will not directly work in this scenario.
Whereas here we have different inputs based on the name which gets populated in the dns yaml file e.g. 

```
- name: "plum"
    ttl: 300
    record: "hmcts-sbox.azurefd.net"
    shutter: true
```

Reply:
or I can do something like this 
${split(".", replace(“${var.cname_records[count.index].name}.${var.zone_name}”, "www.", ""))[0]

let me try that